APK at v3.0.0 was built from non-existing commit

According to the embedded `vcsInfo`, the APK was built from commit ee959ac83de5e81218667635b5bbaf67f21b1bd1 – which is not present here:

<img width="964" height="742" alt="Image" src="https://github.com/user-attachments/assets/a772cb49-f7c2-46a5-a011-89183c4c27f0" />

So we cannot build from it, and hence not confirm whether it is RB or not. As that change is available at the tagged commit, I assume it was squashed – or rather merged from a local branch **after** you built the APK? That is almost certain to break RB, as shown.

But even if I try to build simply from the tagged commit, the build fails:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':common:design:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2402: AAPT: error: style attribute 'attr/windowSplashScreenAnimatedIcon (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimatedIcon)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2403: AAPT: error: style attribute 'attr/windowSplashScreenAnimationDuration (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimationDuration)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2404: AAPT: error: style attribute 'attr/postSplashScreenTheme (aka com.androidvip.sysctlgui.design:attr/postSplashScreenTheme)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:46: AAPT: error: style attribute 'attr/windowSplashScreenAnimatedIcon (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimatedIcon)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:47: AAPT: error: style attribute 'attr/windowSplashScreenAnimationDuration (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimationDuration)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:48: AAPT: error: style attribute 'attr/postSplashScreenTheme (aka com.androidvip.sysctlgui.design:attr/postSplashScreenTheme)' not found.
```

As that's just a verify, I ran `./gradlew assembleRelease -x :common:design:verifyReleaseResources` to see if the build succeeds at least. It builds then, but is not reproducible: the diff between the two APKs is rather huge. For starters:

```diff
     XML ELEM START [lineno=2, name='manifest', #attributes=7]
-      ATTR: http://schemas.android.com/apk/res/android:versionCode=19
+      ATTR: http://schemas.android.com/apk/res/android:versionCode=17
       ATTR: http://schemas.android.com/apk/res/android:versionName='3.0.0'
```

So your APK shows `versionCode: 19`, while [at the tag](https://github.com/Lennoard/SysctlGUI/blob/3.0.0/app/build.gradle.kts#L20) it is 17. Guess we'll have to declare this one failed then – unless you have an idea how to cure it?

That seems to match now, confirmed. Build fails nevertheless:

```
FAILURE: Build failed with an exception.132 actionable tasks: 132 executed
Configuration cache entry stored.


* What went wrong:
Execution failed for task ':common:design:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2402: AAPT: error: style attribute 'attr/windowSplashScreenAnimatedIcon (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimatedIcon)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2403: AAPT: error: style attribute 'attr/windowSplashScreenAnimationDuration (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimationDuration)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values/values.xml:2404: AAPT: error: style attribute 'attr/postSplashScreenTheme (aka com.androidvip.sysctlgui.design:attr/postSplashScreenTheme)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:46: AAPT: error: style attribute 'attr/windowSplashScreenAnimatedIcon (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimatedIcon)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:47: AAPT: error: style attribute 'attr/windowSplashScreenAnimationDuration (aka com.androidvip.sysctlgui.design:attr/windowSplashScreenAnimationDuration)' not found.
         
     ERROR: /build/repo/common/design/build/intermediates/merged_res/release/mergeReleaseResources/values-night-v8/values-night-v8.xml:48: AAPT: error: style attribute 'attr/postSplashScreenTheme (aka com.androidvip.sysctlgui.design:attr/postSplashScreenTheme)' not found.
```

As before, I can try disabling that gradle task – but I'm afraid the result won't be RB then anyway. Did you maybe forget to check in some files?

--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -17,8 +17,8 @@ android {
         applicationId = AppConfig.appId
         minSdk = AppConfig.minSdkVersion
         targetSdk = AppConfig.targetSdkVersion
-        versionCode = 21
-        versionName = "3.0.2"
+        versionCode = 22
+        versionName = "3.0.3"
         vectorDrawables.useSupportLibrary = true
         androidResources {
             localeFilters += listOf("en", "de", "pt-rBR", "tr")

--- a/common/design/build.gradle.kts
+++ b/common/design/build.gradle.kts
@@ -42,6 +42,7 @@ android {
 
 dependencies {
     implementation(libs.androidx.core.ktx)
+    implementation(libs.androidx.core.splashscreen)
 
     api(platform(libs.androidx.compose.bom))
     api(libs.androidx.ui)

--- a/gradle.properties
+++ b/gradle.properties
@@ -15,8 +15,6 @@ org.gradle.jvmargs=-Xmx1536m
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-# Allow databinding to be incremental:
-android.databinding.incremental=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 

